### PR TITLE
raspi scripts and game clock test

### PR DESCRIPTION
**Pull request by steakhouserodriguez**

### CODE REVIEW

File: raspi/deploy_scripts.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash 
+# 
+# deploy_forseti2.sh 
+#
+# via http://www.hypexr.org/linux_scp_help.php
+
+scp -r /home/ajc/forseti2/forseti2 pi@192.168.1.169:/home/pi/forseti2
```
Comment (nikitakit) [inline raspi/deploy_scripts.sh]: Hardcoded path only works on ajc's machine

Comment (steakhouserodriguez) [inline raspi/deploy_scripts.sh]: I guess changing this to relative paths would work?

File: raspi/lcm_build.sh [hunk 2]
```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+# 
+# rasdeploy.sh
```
Comment (nikitakit) [inline raspi/lcm_build.sh]: wrong name here

File: src/pygametext.py [hunk 3]
```diff
@@ -0,0 +1,45 @@
+
```
Comment (nikitakit) [inline src/pygametext.py]: This obviously isn't a game clock.

Can you add a comment to the top of the file to explain a bit about it?

Also, is it launched manually, or via a script?

File: src/pygametext.py [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+#    window = pygame.display.set_mode((1280, 1024))
+    pygame.mouse.set_visible(False)
+    headerfont = pygame.font.SysFont("freesans", 100)
+    font = pygame.font.SysFont("freesans", 250)
+
+    def draw_text(surface, font, text, pos, color):
+    	label = font.render(text, 1, color)
+    	surface.blit(label, pos)
+
+    draw_text(window, font, "Hello, world", (10,10), (255,255,255))
```
Comment (nikitakit) [inline src/pygametext.py]: not useful

File: raspi/deploy_scripts.sh (hunk 1 above)
Comment (nikitakit) [inline raspi/deploy_scripts.sh]: yes, relative paths

File: src/pygametext.py (hunk 4 above)
Comment (nikitakit) [inline src/pygametext.py]: to clarify: the hello world is not useful

File: src/pygametext.py (hunk 3 above)
Comment (steakhouserodriguez) [inline src/pygametext.py]: launched via raspi LXDE autostart

**Pull request merged**